xbmc/utils/XMLUtils.h

```diff
@@ -82,6 +82,7 @@ class XMLUtils
   static void SetStringArray(TiXmlNode* pRootNode, const char *strTag, const std::vector<std::string>& arrayValue);
   static TiXmlNode* SetInt(TiXmlNode* pRootNode, const char *strTag, int value);
   static TiXmlNode* SetFloat(TiXmlNode* pRootNode, const char *strTag, float value);
+  static TiXmlNode* SetDouble(TiXmlNode* pRootNode, const char *strTag, double value);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types